virtme/guest/virtme-init

```diff
@@ -57,19 +57,20 @@ fi
 
 # Adjust max limit of open files
 if [[ -n "${nr_open}" ]]; then
-    echo ${nr_open} > /proc/sys/fs/nr_open
+    printf -- '%s\n' "${nr_open}" > /proc/sys/fs/nr_open
 fi
 
 # devtmpfs might be automounted; if not, mount it.
 if ! grep -q devtmpfs /proc/mounts; then
     # Ideally we'll use devtmpfs (but don't rely on /dev/null existing).
     if [[ -c /dev/null ]]; then
-	mount -n -t devtmpfs -o mode=0755,nosuid,noexec devtmpfs /dev \
-	    &>/dev/null
+	    mount -n -t devtmpfs -o mode=0755,nosuid,noexec devtmpfs /dev \
```

Comment:
The indentation looks strange here and below, no? Maybe better to use spaces here and not a mix of both.

Reply:
Could be, in emacs it looked fine :) But I will probably reformat the whole code base using `shfmt`.